### PR TITLE
Show how to split on a composite primary key

### LINKED ISSUE #2769: SPLIT AT Doc Update
Re: [SPLIT AT](https://cockroachlabs.com/docs//v2.0/split-at.html)

The docs say that SPLIT AT takes a `select statement` but that's not enough info. For example, what would be required to split at a table that has a composite primary key?

According to @jordanlewis we use the syntax `alter table .. split at values ((string, uuid), (string2, uuid2))`

We should make this clear -- I think this may come up when we release partitioning since it uses compound keys.

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes #2769

### CODE REVIEW

**Comment by nstewart** [comment]
Thanks LGTM

**Comment by RaduBerinde** [comment]
Maybe worth mentioning that you can also split at prefix of the PK columns. In this case, you could split at /3 (which means that there is a split before all keys that start with `3`)

**Comment by rmloveland** [comment]
> Maybe worth mentioning that you can also split at prefix of the PK columns. In this case, you could split at /3 (which means that there is a split before all keys that start with 3)

Fixed in latest push - thanks!

File: v2.0/split-at.md [hunk 1]
```diff
@@ -103,6 +103,87 @@ the ranges that store tables or indexes:
...
++-----------+---------+----------+----------+--------------+
+| NULL      | /5/1    |      151 | {2,3,5}  |            5 |
+| /5/1      | /5/2    |      152 | {2,3,5}  |            5 |
+| /5/2      | /5/3    |      153 | {2,3,5}  |            5 |
+| /5/3      | NULL    |      154 | {2,3,5}  |            5 |
++-----------+---------+----------+----------+--------------+
+(4 rows)
+~~~
+
+Alternatively, you could instead split at a prefix of the primary key columns.  For example, to add a split before all keys that start with `3`, run:
```
Comment (jseldess) [inline v2.0/split-at.md]: nits: Don't need both `alternatively` and `instead`. I'd remove `instead`. Also, use just one space following the period.

File: v2.1/split-at.md [hunk 2]
```diff
@@ -103,6 +103,87 @@ the ranges that store tables or indexes:
...
++-----------+---------+----------+----------+--------------+
+| NULL      | /5/1    |      151 | {2,3,5}  |            5 |
+| /5/1      | /5/2    |      152 | {2,3,5}  |            5 |
+| /5/2      | /5/3    |      153 | {2,3,5}  |            5 |
+| /5/3      | NULL    |      154 | {2,3,5}  |            5 |
++-----------+---------+----------+----------+--------------+
+(4 rows)
+~~~
+
+Alternatively, you could instead split at a prefix of the primary key columns.  For example, to add a split before all keys that start with `3`, run:
```
Comment (jseldess) [inline v2.1/split-at.md]: Same as above.

File: v2.0/split-at.md (hunk 1 above)
Comment (rmloveland) [reply v2.0/split-at.md]: Thanks, fixed.

File: v2.1/split-at.md (hunk 2 above)
Comment (rmloveland) [reply v2.1/split-at.md]: Fixed.

**Pull request merged**